AcadeMeData/tests.py

```diff
@@ -1,35 +1,40 @@
 import pytest
-from models import User
+from AcadeMeData.models import User
 
 
-class TestUserModel(pytest):
+class TestUserModel:
     def user_example(self):
-        user_data = {'username': "username", 'password': "password", 'email': "user@example.com", 'type': "S",
-                     'university': "RU",
-                     'degree': "CS"}
-        user = User.create_user(*user_data)
-        return user, user_data
+        data_for_user = {'username': "username", 'password': "password", 'email': "user@example.com", 'type': "S",
+                         'university': "RU",

```

Comment:
please delete spaces, it will give you flake8 error

Reply:
when the line is too big this is the only way it pass flake8